runtime/makelib/targets.mk.zos.inc.ftl

```diff
@@ -112,6 +112,14 @@ ifeq ($(UMA_TARGET_NAME),jvmti_test_src)
   UMA_ZOS_FLAGS += -Wc,DLL,EXPORTALL
 endif
 
+COMMA := ,
+
+# Compile jniargtestssystemlink with non-XPLINK (system) linkage
+ifeq ($(UMA_TARGET_NAME),jniargtestssystemlink)
+  UMA_ZOS_FLAGS := $(subst xplink$(COMMA),,$(UMA_ZOS_FLAGS))
+  UMA_LINK_FLAGS := $(subst xplink$(COMMA),,$(UMA_LINK_FLAGS))
```

Comment:
These would be better if the comma starts the pattern to handle `-Wl,xplink`:
```
UMA_LINK_FLAGS := $(subst $(COMMA)xplink,,$(UMA_LINK_FLAGS))
```

Reply:
Right, I wanted to do that as well but there is an `xplink` option that is preceeded by a double quote which would not have been caught. I do agree that replacing `,xplink` is much better, so I've gone ahead and just moved that option to the left so it follows the pattern.

Fixed in aa851c4 (see [force push](https://github.com/eclipse/openj9/compare/5f938ef88bb3f8d33239c8a3c05657370baa457e..ab713cac6f91859325f0e62136936b0bb7ddbec8)).

[1] https://github.com/eclipse/openj9/blob/84db769ba38ffc9db90691fa80fd47c98115a04c/runtime/makelib/targets.mk.zos.inc.ftl#L72-L73